cmd/erasure-object.go

```diff
@@ -614,8 +616,8 @@ func (er erasureObjects) putObject(ctx context.Context, bucket string, object st
 				parityDrives++
 			}
 		}
-		if ecOrg != parityDrives {
-			opts.UserDefined[xhttp.MinIOErasureUpgraded] = fmt.Sprintf("%d->%d", ecOrg, parityDrives)
+		if parityOrig != parityDrives {
+			opts.UserDefined[minIOErasureUpgraded] = strconv.Itoa(parityOrig) + "->" + strconv.Itoa(parityDrives)
```

Comment:
also the main reason to move to this style is because of this

```
go test -bench .
goos: linux
goarch: amd64
pkg: github.com/minio/minio/test1
cpu: Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
BenchmarkFmt-4           9585319               123.6 ns/op             4 B/op          1 allocs/op
BenchmarkStr-4          37062789                31.80 ns/op            0 B/op          0 allocs/op
PASS
ok      github.com/minio/minio/test1    2.528s
```

Reducing the usage of `fmt.Sprintf` in the hot call path is removed.